en/mkdocs.yml

```diff
@@ -287,6 +287,9 @@ nav:
           - 'Setting Up ReCaptcha': learn/setting-up-recaptcha.md
         - 'Workflows': 
           - 'Workflows': learn/workflow-management.md
+          - 'Adding a workflow Engine':
+            - 'Adding a workflow Engine': learn/adding-a-workflow-engine.md
+            - 'Configuring the BPM Profile as a Workflow Engine': learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine.md
```

Comment:
'Configuring BPM as a Workflow Engine' ?

Reply:
BPM has been deprecated now. So I feel that emphasizing that it is only a profile helps.